auth/user_mgt.go

```diff
@@ -56,6 +56,19 @@ type UserInfo struct {
 	UID        string `json:"rawId,omitempty"`
 }
 
+// MultiFactorInfo describes a user enrolled second factor.
+type MultiFactorInfo struct {
```

Comment:
This API should closely mirror the type signatures defined for Node.js:

```go
type MultiFactorInfo struct {
  UID string
  DisplayName string
  EnrollmentTimestamp int64
  FactorID string
}
```

You will probably need to define a multiFactorInfoQuery internal type for unmarshalling, and then initialize this type based on that.

Reply:
Ah, I must've halfway through accidentally switched to mapping names to the REST API in stead of the Node.js API. I see. I've had a look at how the Node.js library converts the REST response JSON to the internal structures. I'll fix this early next week!

Reply:
Mental note: https://github.com/firebase/firebase-admin-node/blob/f624b2963900deebed0fdb5cbe66ce7eb0d244b8/src/auth/user-record.ts

Reply:
Instead of what you proposed `multiFactorInfoQuery`, I've sticked with `MultiFactorInfoResponse`, which makes more sense to me and is in line with the Node.js naming.